src/WebJobs.Script.WebHost/WebScriptHostExceptionHandler.cs

```diff
@@ -48,14 +49,14 @@ public async Task OnTimeoutExceptionAsync(ExceptionDispatchInfo exceptionInfo, T
             // Give the manager and all running tasks some time to shut down gracefully.
             //await Task.Delay(timeoutGracePeriod);
 
-            _jobHostEnvironment.Shutdown();
+            _applicationLifetime.StopApplication();
```

Comment:
While you're in here, can you just remove all the TODO comments above? @fabiocav can confirm.